scripts/uberenv/packages/conduit/package.py

```diff
@@ -34,16 +34,18 @@ class Conduit(Package):
     coupling between packages in-core, serialization, and I/O tasks."""
 
     homepage = "http://software.llnl.gov/conduit"
+    url      = "https://github.com/LLNL/conduit/releases/download/v0.3.0/conduit-v0.3.0-src-with-blt.tar.gz"
     git      = "https://github.com/LLNL/conduit.git"
 
     version('master', branch='master', submodules=True, preferred=True)
```

Comment:
Is this meant to be ``axom``'s version of ``conduit/package.py``, or is it going back to ``spack``?

Asking b/c I don't think ``conduit@master`` should be ``preferred`` for ``axom``.

Reply:
Package versions are locked down in scripts/uberenv/spack_configs/$SYS_TYPE/packages.py

Reply:
Yes, -- they're locked down for our LC configs (i.e. for our internal development).
But presumably other linux or mac ``uberenv`` users would get ``conduit@master`` rather than the tested version (i.e. currently ``conduit@0.5.1``), which could be problematic.

In general, it hasn't been too bad since conduit has been really good about backwards compatibility, but some of our users have had issues with changing blueprint schemas between versions, where, e.g. ``blueprint::verify`` started to fail.

Reply:
Also this isnt our package its cyrus's and is consistent with what is in the repo